packages/experimental-package/src/components/RemoveModal/RemovalModal.js

```diff
@@ -0,0 +1,85 @@
+import React, { useState } from 'react';
+import { Modal, TextInput } from 'carbon-components-react';
+import PropTypes from 'prop-types';
+import { expPrefix } from '../../global/js/settings';
+
+export const RemovalModal = ({
+  body,
```

Comment:
Can we not repeat Carbon props somehow?

Reply:
Raised this issue to discuss how we create shallow wrappers around Carbon components

https://github.com/carbon-design-system/ibm-cloud-paks/issues/138